web_ui/frontend/app/registry/page.tsx

```diff
@@ -102,35 +99,37 @@ export default function Home() {
                     </Collapse>
                 </Grid>
                 <Grid item xs={12} lg={8} justifyContent={"space-between"}>
-                    <UnauthenticatedContent>
-                        <Box>
+                    { user == undefined || !user.authenticated &&
+                        <Alert severity="info">
                             <Typography variant={"body1"}>
                                 Login to register new namespaces.
-                                <Button sx={{ml:2}} variant={"contained"} size={"small"} color={"primary"} href={"/view/login/"}>Login</Button>
+                                <Link href={"/login/"}>
```

Comment:
Is it possible to have a redirect query parameter here to redirect user back to registration page?